Dockerfile

```diff
@@ -14,7 +14,7 @@ WORKDIR /go-quai
 RUN env GO111MODULE=on go run build/ci.go install ./cmd/go-quai
 
 # Stage 2
-FROM golang:1.19-alpine
+FROM golang:1.20-alpine
```

Comment:
this needs to happen in both stages of the docker container and then it is ready